modules/OsmOsisManager.py

```diff
@@ -297,7 +297,7 @@ def init_diff(self, conf):
         elif line.startswith("maxInterval"):
             if "geofabrik" in conf.download["diff"]:
                 # on daily diffs provided by Geofabrik, we should apply only one diff at a time
-                sys.stdout.write("maxInterval=" + str(60*60*24/2)) # 1/2 day at most
+                sys.stdout.write("maxInterval=" + str(round(60*60*24/2))) # 1/2 day at most
             else:
                 sys.stdout.write("maxInterval=" + str(7*60*60*24)) # 7 day at most
```

Comment:
Better use **//**
as per https://www.python.org/dev/peps/pep-0238/

    $ docker run -ti python:3.7 bash
    root@ff315ac86108:~# echo 'print("maxInterval=" + str(60*60*24/2))' | python
    maxInterval=43200.0
    root@449e4ea1e59c:/# echo 'print("maxInterval=" + str(60*60*24//2))' | python
    maxInterval=43200

Reply:
OK, I'm fine with this version too.

Reply:
Please, change the commit to `//`

Reply:
Done